deployment-ctx/11-environment-variables.sh

```diff
@@ -48,6 +47,9 @@ if [ ! -f $SCRIPT_CACHE_FILE ]; then
 apiUrl: '${API_URL_RESOLVED%/}',
 mlUiUrl: '${ML_UI_URL_RESOLVED}',
 mlNotebooksUrl: '${ML_NOTEBOOKS_URL_RESOLVED}',
+oidcAuthUrl: '${OIDC_AUTH_URL}',
```

Comment:
It looks like we'll also need to manage the OIDC auth url in the same loop in "Step 3" where we add it to Content Security Policy. I need to rewrite step 3 into a function, so I can take care of that.

We shouldn't need to add the redirect URL to CSP, since that should only be read by the OIDC authenticator, correct?